posthog/migrations/0147_event_props_fuzzy_search.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.1.8 on 2021-04-27 01:13
+
+import django.contrib.postgres.indexes
+from django.contrib.postgres import operations
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0146_eventproperty_sync"),
+    ]
+
+    operations = [
+        operations.TrigramExtension(),
+        migrations.AddIndex(
+            model_name="eventdefinition",
+            index=django.contrib.postgres.indexes.GinIndex(
```

Comment:
Would be great to have someone with a ton more expertise than me in Postgres to take a look at these indices. I'll do some performance testing anyways to see how useful they are, but still worth taking a look.